frontend/viewer/src/lib/services/idle-service.ts

```diff
@@ -0,0 +1,23 @@
+import {IsIdle} from 'runed';
+import {getContext, hasContext, setContext} from 'svelte';
+
+const symbol = Symbol.for('fw-lite-idle-service');
+const DEFAULT_IDLE_TIMEOUT_MS = 5 * 60 * 1000;
+
+export function useIdleService(timeout?: number): IdleService {
```

Comment:
I think we talked about this in office, let's scrap the timeout option as it will only work the first place that runs which is not under our control.

Reply:
Done in commit 36baee1eb.